content/posts/pandoc/pandoc.md

```diff
@@ -0,0 +1,334 @@
+
+To create a slide shows with Pandoc is quite simple thus the basic syntax is Markdown which is a familiar language with all developers or computer scientist. 
+
+Let's create some slide with Pandoc:  🎉🎉🎉
+
+Create a `main.md` file that contains:
+
+```markdown
+# Section heading 1 slide
+## Slide 1 of Section 1 
+- This is a list 
+- Another list item
+- etc,...
+
+![Image example](testImage.png)
+
+# Section heading 2
+...
+```
+
+Once you have pandoc and latex (with packages to run beamer) installed, you just have to use the following command to compile the presentation:
+```bash
+pandoc main.md -t beamer -o output.pdf
+```
+
+![](/posts/pandoc/images/compileFlow.png)
+
+Pandoc converted *main.md* into a beamer latex file main.tex and after that they use the default way (`pdflatex`) to transfer beamer *.tex to a Presentation file. By default, pandoc will use `slide-level=2`, it means that level 1 markdown headings represents section headings - as the picture below ( line preceded with a single hash `# Section heading 2`). And Level 2 headings represented a new slide with the title name. ( line preceded with double hash `## Slide 1 of Section 1`). So that the headings above the defined number in the hierarchy are used to divide the slide show into sections; the headings below the defined number create subheads within a slide. You might want to read more about [Structuring the slide show.](https://pandoc.org/MANUAL.html#structuring-the-slide-show)
+
+**Code explaination**
+
+- The first line adds a section tittle. This is not part of the slide, but can be used to generate table of contents, and in slide navigation.
+- The second line creates a new slide with the slide tittle.
+- To create a `itemize` as LaTeX, we can you the same method in markdown.
+- Also we can use the same method in markdow to create a figure, table inside a slide.
+
+**Result** 
+
+![](/posts/pandoc/images/result1.png)
+
+**Code comparation with LaTeX**
+
+```latex
+\documentclass[t]{beamer}  
+\setbeamertemplate{navigation symbols}{}
+\begin{document}
+\section{Section heading 1 slide}
+\begin{frame}
+\frametitle{Slide 1 of Section 1}
+\begin{itemize}
+  \item {
+   	This is a list 
+  }
+  \item {
+  	Another list item
+  }
+  \item{
+  	etc,...
+  }
+\end{itemize}
+\begin{figure}
+  \includegraphics{testImage.png}
+\end{figure}
+\end{frame}
+\section{Section 2}
+\end{document}
+```
+
+&rarr; Markdown is muchhhhhhhhh shorter line of code comparing to LaTeX with the same content.
+## Advanced configuration
+
+#### YAML
+You might want a slide contain a Beamer's theme, author name, presentation name,... - a fully features as LaTeX beamer but written in markdown. To deal with that, we can add some YAML configuration at the beginning of the markdown file.\
```

Comment:
- a slide *containing* .. 
- a Beamer's theme, *an* author name, *a* presentation name, ...
- ~~a fully features as LaText beamer~~ all of LaTex beamer's features ...